lib/verify/response.rb

```diff
@@ -1,4 +1,5 @@
 module Verify
+  class MissingResponseAttribute < StandardError; end
```

Comment:
Define this inside the `Response` class to avoid weirdness with the autoloading and I think this is good to go 👍🏽

Reply:
Done 👍